CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.0.2 (2019-04-22)
+
+- Updated modal component to prevent clicks from propagating. [#35](https://github.com/blackbaud/skyux-modals/pull/35)
```

Comment:
Should be prefixed with "Fixed"?
Something like? "Fixed modal component to stop click events from bubbling beyond its host component."

Reply:
I wasn't 100% confident this was a bug with modals. It certainly helps fix a downstream bug though. Nothing with "modals" was necessarily broken here.